docs/DeveloperGuide.adoc

```diff
@@ -150,97 +185,132 @@ The `Storage` component,
 The following activity diagram summarizes what happens when a user executes a new command:
 
-image::CommitActivityDiagram.png[]
+image::ImportActivityDiagram.png[]
+
+
+=== [Coming soon in v1.3] Export
+==== Implementation [Proposed]
+The Export command gets the scheduled time slots from the Model and writes them in the specified .csv file.
+CsvWriter facilitates the writing to the specified file.
+
+* CsvWriter makes use of `BufferedWriter` to write data into the specified file.
+
+Below shows the sequence diagram of an example export command.
+
+image::ExportSequenceDiagram.png[]
+
+The implementation is very similar to the Import feature. The only differences are in the Model where CsvWriter gets
+the scheduled time slots from the Model and proceeds to write it into the specified file using a `BufferedWriter`.
 
-==== Design Considerations
+The Activity Diagram below summarises the execution of the export command.
 
-===== Aspect: How undo & redo executes
+image::ExportActivityDiagram.png[]
 
-* **Alternative 1 (current choice):** Saves the entire address book.
-** Pros: Easy to implement.
-** Cons: May have performance issues in terms of memory usage.
-* **Alternative 2:** Individual command knows how to undo/redo by itself.
-** Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
-** Cons: We must ensure that the implementation of each individual command are correct.
 
-===== Aspect: Data structure to support the undo/redo commands
+=== [Coming soon in v1.3] Email
+==== Implementation
 
-* **Alternative 1 (current choice):** Use a list to store the history of address book states.
-** Pros: Easy for new Computer Science student undergraduates to understand, who are likely to be the new incoming developers of our project.
-** Cons: Logic is duplicated twice. For example, when a new command is executed, we must remember to update both `HistoryManager` and `VersionedAddressBook`.
-* **Alternative 2:** Use `HistoryManager` for undo/redo
-** Pros: We do not need to maintain a separate list, and just reuse what is already in the codebase.
-** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
-// end::undoredo[]
+The Email feature makes use of the `java.awt.Desktop` package to activate the default Mail client of the user.
 
-// tag::dataencryption[]
-=== [Proposed] Data Encryption
+* The `To:` field is automatically populated with all the emails that are tagged to a particular Interviewee.
+* The `Cc:` field is configurable by the user via an optional configuration file.
+* The subject and message body are also automatically populated with details that are relevant to the Interviewee, depending on the context of the command used.
 
-_{Explain here how the data encryption feature will be implemented}_
+image::EmailSequenceDiagram.png[]
```

Comment:
1. intervieweeName and intervieweeToEmail should have () to signify it is a method
2. Constructor method names must be included